content/docs/user-guide/assets/scene-settings/blender-scene-exporter.md

```diff
@@ -0,0 +1,9 @@
+---
```

Comment:
May be able to remove Blender scene export from this PR - check in with me and @HogJonny-AMZN about our current alternatives.

Reply:
removed